### ISSUE TITLE
﻿plex_library_section_size_count depth limited and albums/tracks ignored

### ISSUE THREAD

**Issue by AndreiGavriliu** [arnarg/plex_exporter#5]
Hi,

I just implemented this `plex_exporter` in my `OKD` environment and wanted to see how many artists/albums/tracks I have in my library. I noticed it only counts the `artist` type, ignoring the `album` and `track`

Also, when checking how many Photos I have, it only counts the first level of photo albums. My folder structure is as follows:
```
2018/2018-04-12/Some Event/image0005.jpg
2018/2018-08-04/Some other Event/image0004.jpg
2018/2018-08-04/Some other Event/image0008.jpg
2019/2019-09-12/Someone's Birthday/image0085.jpg
```
This structure only reports 2 photos - one for each year.

Is it possible to change the depth of the photo albums and also add albums and tracks counts?

If it helps, here is a trace output from my `plex_exporter` pod:

```
time="2020-01-16T18:40:40Z"
level=trace 
msg="{
  cfa719f1b4dbde85825191ee039df677b2509deb Plex 1.18.4.2171-ac2afe5f8 Linux 0 
  [
    {Concerts movie 74} 
    {Documentaries movie 31} 
    {Kids movie 138}
    {Movies movie 283} 
    {Music artist 258} 
    {Photos photo 11} 
    {TV Shows show 12}
  ]
}" 
context=collector
```

Thanks,
Andrei

**Comment by arnarg** [comment]
Yeah I noticed the same thing with TV shows when I was writing this exporter. You get a TV shows library with a number of TV shows but you have to dig in deeper if you want to get the number of episodes.

For simplicity I just grabbed the Libraries and showed their size, as I'm most interested in movies anyway. There has to be created a specific logic for each Library type.

To be perfectly honest, I don't need this feature so I'm not sure when I can find the time to fix this. But I accept contributions.
